templates/repo/projects/view.tmpl

```diff
@@ -74,7 +74,7 @@
 
 			<div class="ui segment board-column">
 				<div class="board-column-header">
-					<div class="ui large label board-label">{{.Title}}</div>
+					<div class="ui large label board-label">{{.Title}}{{if .Default}} <div class="ui label">Default</div>{{end}}</div>
```

Comment:
needs to be i18n'd

Will need a new key in locale_en-US.ini too

Reply:
-> https://github.com/go-gitea/gitea/pull/14147#discussion_r548890234